notebooks/en/llm_judge_evaluating_ai_search_engines_with_judges_library.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
<p>Maybe clarify <code>MTBenchChatBotResponseQuality</code> is also a "grader" type of judge (not really clear right now). It can say something like "Response Quality Evaluation Grader" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Done!</p>